### ISSUE TITLE
[FEAT]: Pass favicon of provider downstream

### ISSUE THREAD

**Issue by Rouzax** [Prowlarr/Prowlarr#589]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

Currently, when adding Prowlarr indexers to Medusa it shows with the favicon of Prowlarr.
![image](https://user-images.githubusercontent.com/4103090/140889554-491ebcac-5b71-4384-beb8-ed7f48a49839.png)

### Describe the solution you'd like

Would it be possible that Prowlarr also pushes the favicon of the indexer downstream?

### Describe alternatives you've considered

No real other solution

### Anything else?

Would look something like this, which would help with distinguishing between providers.
![image](https://user-images.githubusercontent.com/4103090/140889797-1009fb8b-dcbc-486e-a7c8-45ce3b27437d.png)

**Comment by Qstick** [comment]
This is a Medusa integration and not a Prowlarr integration. We don't natively support Medusa from the Prowlarr side as of yet. 

That being said we can look at passing it through but Medusa will have to change their implementation to use it

**Comment by Rouzax** [comment]
There is a feature Request open at Medusa, and they are willing to look at it.
https://github.com/pymedusa/Medusa/discussions/10059

**Comment by neo-neo1** [comment]
I brought this up a month ago. I agree it's a nice feature to have that's currently missing. 

- https://github.com/pymedusa/Medusa/discussions/9893

**Comment by p0psicles** [comment]
Yeah but I prefer getting an icon from prowlarr. So if they add something, I'll add it

**Comment by p0psicles** [comment]
Hi guys. I know yall busy. But maybe an update on this?

**Comment by Qstick** [comment]
I have a branch local that sorts this, just need time to give it some love
